### ISSUE TITLE
Unknown loss function for Ranking

### ISSUE THREAD

**Issue by Jbiloki** [eclipse/deeplearning4j#7895]
I am looking to implement a model that uses a custom tensorflow loss function defined here:  [https://github.com/tensorflow/ranking/blob/master/tensorflow_ranking/python/losses.py](https://github.com/tensorflow/ranking/blob/master/tensorflow_ranking/python/losses.py)

which is blocked by:

Caused by: org.deeplearning4j.nn.modelimport.keras.exceptions.UnsupportedKerasConfigurationException: Unknown Keras loss function _pairwise_logistic_loss

**Comment by maxpumperla** [comment]
@Jbiloki unfortunately, we don't support custom losses as of now. you'd have to implement that in dl4j first, as it's not an official keras loss.

**Comment by Jbiloki** [comment]
@maxpumperla Thank you for the update, adding it myself is a fine option. How can I register it as a custom loss function to use for the imported model once created?

**Comment by maxpumperla** [comment]
@Jbiloki that's a good point. we have a way of registering custom layers, but not custom losses.

I'll have to look into that first.
